enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -170,7 +172,19 @@
     // LDAP authorization settings
     //-----------------------------------------------------
 
-    @Description( "Perform LDAP search for authorization info using a system account." )
+    @Description( "Perform LDAP search for authorization info using a system account or the user's own account.\n\n" +
+                  "If this is set to `false` (default), the search for group membership will be performed " +
+                  "directly after authentication using the ldap context bound with the user's own account. " +
+                  "The mapped roles will be cached for the duration of `dbms.security.auth_cache_ttl`, " +
+                  "and then expire, requiring re-authentication. To avoid frequently having to re-authenticate " +
+                  "sessions you may want to set a relatively long auth cache expiration time together with this " +
+                  "option. NOTE: This option will only work if the users are permitted to search for the " +
+                  "group membership attributes on themselves in the directory.\n\n" +
```

Comment:
"for the group membership attributes on themselves" => "for their own group memberships" or "for group memberships for their own user"